docs/src/05-mgx-taxonomic-profiles.jmd

```diff
@@ -274,31 +273,40 @@ savefig(joinpath(figures, "taxonomic-profiles-kids-shannon.svg"));
 Let's look at the phyla too:
 
 ```julia
-
 pkids = view(pabt, sites=firstkids(resolve_sampleID.(samplenames(abt))))
 
 kids_bact = vec(collect(occurrences(view(pkids, species=occursin.("Bact", speciesnames(pkids))))))
 kids_firm = vec(collect(occurrences(view(pkids, species=occursin.("Firm", speciesnames(pkids))))))
 kids_act = vec(collect(occurrences(view(pkids, species=occursin.("Actino", speciesnames(pkids))))))
 kids_proteo = vec(collect(occurrences(view(pkids, species=occursin.("Proteo", speciesnames(pkids))))))
```

Comment:
<img width="1006" alt="Screen Shot 2019-07-08 at 12 02 13 PM" src="https://user-images.githubusercontent.com/30243213/60825186-211a5780-a179-11e9-98b5-3968c4dd35ea.png">
This code prints this output. Is it supposed to do this? If so, unclear what it means.